en/views.rst

```diff
@@ -460,7 +462,7 @@ Contacts plugin::
     {
         public function view_active()
         {
-            $this->layout = 'Contacts.contact';
+            $this->getView()->layout('Contacts.contact');
         }
```

Comment:
Yes it could confuse people. The alternative would be to check that view instance is of the same class as $viewClass on every call to getView().

Using public properties instead of getter/setter comes to bite us :)